### ISSUE TITLE
Smooth Scroll Fails to Move Keyboard Focus and Breaks Accessibility

### ISSUE THREAD

**Issue by amberhinds** [brainstormforce/astra#5443]
### Describe the bug:
When smooth scroll is turned on in the theme, it visually scrolls the page but fails to shift the keyboard focus to the targeted area. This is a major accessibility problem that makes the "skip to content" links and any other jump links or anchor links on the site utterly unuseable for screen reader and keyboard-only users.

### To reproduce:
1. Turn on smooth scroll.
2. On the front end of the website, tab to the skip to content link or another anchor link.
3. Hit return to trigger the link.
4. The website will visually scroll.
5. Hit the tab key again to see that your keyboard focus is still up higher on the page than where the website visually scrolled to.

### Expected behavior:
When an anchor link is clicked, the keyboard focus should shift to the targeted element.

### Screenshots:
Here's a video I made that shows the current (incorrect) behavior and the expected behavior: https://youtu.be/Po1cfpKJSaw

### Fix Recommendation
[This CSS Tricks article](Here's a video I made that shows the current (incorrect) behavior and the expected behavior: https://youtu.be/Po1cfpKJSaw) might be a helpful resource for how you can modify your Smooth Scroll JavaScript to shift focus appropriately.

Also reference the [focus() method docs.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus)

### Isolating the problem:

- [x] This bug happens with no other plugins activated
- [x] This bug happens with The Astra theme active
- [x] I can reproduce this bug consistently using the steps above

### WordPress version:
6.3.1

### Astra Theme version:
4.3.1

**Comment by thebengalboy** [comment]
Hi @amberhinds 

Thank you for bringing this issue to our attention. We really appreciate your help.

Accessibility is one of our top priorities, and we are committed to resolving this issue with priority. We will verify the issue and release a fix as soon as possible.

We will keep you updated on our progress. In the meantime, please let us know if you have any other questions or concerns.

Thanks,
Mehedi Hasan, from Astra team.

**Comment by Sudhanshu-1508** [comment]
Hi @amberhinds 

Thank you again for bringing this issue to our attention.

We are delighted to inform you that the issue you reported has been thoroughly investigated and successfully resolved in our latest Astra 4.4.1 update.

Please let us know if you face any other similar issues. I’m closing this issue considering this has been fixed. Feel free to contact us if necessary. 

Thanks, 
Sudhanshu Kulkarni, from Astra Team.

**Comment by amberhinds** [comment]
That's great @Sudhanshu-1508! Thank you!
